libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -150,4 +156,30 @@ private void migrateAnalyticsData() {
             }
         }
     }
+
+    private void upgradeTo9Dot2() {
```

Comment:
Upgrade steps.

Reply:
Add `TODO:` here to remove this  in `Mobile SDK 10.0`.